detections/endpoint/linux_auditd_magic_system_request_key.yml

```diff
@@ -0,0 +1,79 @@
+name: Linux Magic SysRq Key Abuse
+id: 22c03600-f84a-47fa-abaa-ffbe3e72c782
+version: 1
+date: '2025-08-28'
+author: Milad Cheraghi
+status: production
+type: TTP
+description: |
+  Detects potential abuse of the Linux Magic SysRq (System Request) key by adversaries
+  with root or sufficient privileges to manipulate or destabilize a system.
+  Writing to /proc/sysrq-trigger can crash the system, kill processes, or bypass standard logging.
+  Monitoring SysRq abuse helps detect stealthy post-exploitation activity.
+data_source:
+  - Linux Auditd Path
+search: '`linux_auditd` type=Path name="/proc/sysrq-trigger" OR name="/proc/sys/kernel/sysrq" OR name="/etc/sysctl.conf"
+  | rename host as dest
+  | stats count min(_time) as firstTime max(_time) as lastTime by dest name
+  | `security_content_ctime(firstTime)`
+  | `security_content_ctime(lastTime)`
+  | `linux_auditd_sysrq_abuse_filter`'
```

Comment:
I would consider checking the output of the `build` Git action Job on this CI and making the appropriate fixes . 
 
IN this case the macro should`linux_macro_sysrq_abuse_filter`